frontend/index.html

```diff
@@ -5,9 +5,9 @@
   <meta charset="utf-8" />
   <meta name="viewport" content="width=device-width, initial-scale=1" />
   <meta name="theme-color" content="#fff" />
-  <link rel="shortcut icon" type="image/ico" href="build-files/shop/icon.png" />
+  <link rel="shortcut icon" type="image/ico" href="build-files/store/icon.png" />
   <script src="/webxdc.js"></script>
-  <title>Webxdc Shop</title>
+  <title>Webxdc Store</title>
```

Comment:
title usage is discouraged in WebXDC docs since it is not displayed, I think this can be removed completely, one less thing to rename later :)

```suggestion
```

Reply:
noted but there are some other things as well -- wanted to just have a dumb-rename without thinking :)